bundle/jsky.app.ot/src/main/scala/jsky/app/ot/gemini/editor/targetComponent/details/SourceDetailsEditor.scala

```diff
@@ -248,18 +256,18 @@ final class SourceDetailsEditor extends GridBagPanel with TelescopePosEditor {
     repaint()
   }
 
-  // enable/disable UI elements as needed
-  private def enableAll(b: Boolean) = {
-    enabled = b
+  def updateEnabledState(b: Boolean): Unit = {
+    val hasProfile = Option(spt).exists(_.getTarget.getSpatialProfile.isDefined)
+    enabled = b && hasProfile
 
-    peer.getComponents.foreach(_.setEnabled(b))
+    peer.getComponents.foreach(_.setEnabled(enabled))
 
-    if (b) {
-      tooltip = ""
-      border = titleBorder("Source")
-    } else {
+    if (!hasProfile && editable) {
       tooltip = "Click on title to activate this."
       border = titleBorder("Source [Click to Activate]")
+    } else {
```

Comment:
@fnussber  I went with the flow here but I'm wondering about the "Click to Activate" idea.  There's no way to go back once you've clicked it and it's a first for this kind of interaction in the OT.   We could certainly simplify the enabled state behavior if the source widgets are just always active when the observation is editable.  That is, it wouldn't need special case handling.

If that's not desirable, another idea would be to introduce a radio button (or something) at the top to give the user the choice of defining the source information or not.  By default it would be left undefined but if they elect to define the source then the spatial profile and spectral distribution would appear?

Either way the widgets wouldn't require special case handling.  If we decide to make any changes, let's merge this PR first regardless and then update the ITC widgets.